core/src/test/java/jenkins/mvn/SettingsProviderHelperTest.java

```diff
@@ -0,0 +1,39 @@
+package jenkins.mvn;
+
+import hudson.AbortException;
+import org.junit.Test;
+
+import java.io.File;
+import java.net.URL;
+
+/**
+ * @author <a href="mailto:cleclerc@cloudbees.com">Cyrille Le Clerc</a>
+ */
+public class SettingsProviderHelperTest {
```

Comment:
Would be nice to have a direct test for the provider with old API implementation. Just to make sure the migration logic does not blow up. Now it **will** blow up if the workspace is null in the build